commands/play.js

```diff
@@ -104,9 +104,12 @@ module.exports = {
 			player.handler = new PlayerHandler(interaction.client, player);
 			player.queue.channel = interaction.channel;
 			await player.connect(interaction.member.voice.channelId, { deafened: true });
-			// that kid left while we were busy bruh
-			if (!interaction.member.voice.channelId) {
-				await interaction.replyHandler.locale('DISCORD_INTERACTION_CANCELED', {}, interaction.user.id);
+			// Ensure that Quaver destroys the player if the user leaves the channel while Quaver is queueing tracks
+			// Ensure that Quaver destroys the player if Quaver gets timed out by the user while Quaver is queueing tracks
+			const timedOut = interaction.guild?.members.me.isCommunicationDisabled();
+			if (!interaction.member.voice.channelId || timedOut) {
+				if (timedOut) await interaction.replyHandler.localeError('DISCORD_BOT_TIMED_OUT');
+				if (!timedOut) await interaction.replyHandler.locale('DISCORD_INTERACTION_CANCELED', {}, interaction.user.id);
```

Comment:
redundant `if (!timedOut)` condition

Reply:
No it is not, for instance Quaver is timed out, it would only trigger `DISCORD_BOT_TIMED_OUT`, not including `DISCORD_INTERACTION_CANCELED`

Reply:
I will apply your suggestion unless if it gives an error.